docs/source/guides/staking_guide.rst

```diff
@@ -1,8 +1,19 @@
+.. _staking-guide:
+
 =======================
 NuCypher Staking Guide
 =======================
 
-All staking-related operations are performed through the ``nucypher stake`` command:
+The account which is actively doing staking operations is necessary to be a hot
+wallet. But it doesn't have to be the same account as the one which receives and
+sends NU tokens. The account which holds NU tokens is called StakeHolder, or
+Staker, and the account which participates in the network as Ursula node is
+called Worker.
+StakeHolder and Worker can have the same Ethereum account, but it's better to
```

Comment:
I'd rephrase this to make it in a less positive light, saying first what is recommended (staker != worker), and only later saying that staker==worker is allowed, but not recommended.